### ISSUE TITLE
Add NCEO Africa dataset to the API

### ISSUE THREAD

**Issue by abarciauskas-bgse** [NASA-IMPACT/cloud-optimized-data-pipelines#128]
Note this ticket is on hold until we get confirmation from Pedro it is ok to publish the file in it's public location.

Supports https://github.com/NASA-IMPACT/cloud-optimized-data-pipelines/issues/115

## Identify the dataset and what the processing needs are
- [x] Identify dataset and where it will be accessed from. 

- [x] Checking with Pedro it can be publicly accessible

There is just one file which is publicly accessible here: s3://nasa-maap-data-store/file-staging/nasa-map/nceo-africa-2017/AGB_map_2017v0m_COG.tif

## Design the metadata and publish to the Dev API

- [x] Checking with Pedro that the following values should be used:

For the collection + item metadata:
* id: "nceo-africa-2017" (for the collection) "AGB_map_2017v0m_COG" for the item
* title: "NCEO Africa Biomass 2017"
* description: "The NCEO Africa Aboveground Woody Biomass (AGB) map for the year 2017 at 100m spatial resolution was developed using a combination of LiDAR, Synthetic Aperture Radar (SAR) and optical based data."
* bounding box: -18.2735295, -35.0540590, 51.8642329, 37.7310386
* spatial extent: 2017

- [ ] Review conventions for generating STAC collection and item metadata:
   - Collections: https://github.com/NASA-IMPACT/delta-backend/issues/29 and STAC version 1.0 specification for collections
   - Items: https://github.com/NASA-IMPACT/delta-backend/issues/28 and STAC version 1.0 specification for items
   - NOTE: The delta-backend instructions are specific to datasets for the climate dashboard, however not all datasets are going to be a part of the visual layers for the dashboard so I believe you can ignore the instructions that are specific to "dashboard" extension, "item_assets" in the collection and "cog_default" asset type in the item
  
- [ ] After reviewing the STAC documentation for collections and items and reviewing existing scripts for generating collection metadata (generally with SQL) and item metadata, generate or reuse scripts for your collection and a few items to publish to the testing API. There is some documentation and examples for how to generate a pipeline or otherwise document your dataset workflow in https://github.com/NASA-IMPACT/cloud-optimized-data-pipelines. We would like to maintain the scripts folks are using to publish datasets in that repo so we can easily re-run those datasets ingest and publish workflows if necessary.

- [ ] If necessary, request access and credentials to the dev database and ingest and publish to the Dev API. Submit a PR with the manual or CDK scripts used to run the workflow to publish to the Dev API and include links to the published datasets in the Dev API

## Publish to the Staging API

Once the PR is approved, we can merge and publish those datasets to the Staging API

**Comment by abarciauskas-bgse** [comment]
I was able to ingest and publish using https://github.com/NASA-IMPACT/cloud-optimized-data-pipelines/pull/130/

**Comment by abarciauskas-bgse** [comment]
Also opened https://github.com/NASA-IMPACT/cloud-optimized-data-pipelines/pull/135 to include read access to nasa-maap-data-store

**Comment by abarciauskas-bgse** [comment]
This is complete, although the PR is outstanding https://github.com/NASA-IMPACT/cloud-optimized-data-pipelines/pull/130
